packages/melos/lib/src/command_runner/init.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+
+import '../../melos.dart';
+import '../common/utils.dart';
+import 'base.dart';
+
+class InitCommand extends MelosCommand {
+  InitCommand(super.config) {
+    argParser.addOption(
+      'directory',
+      abbr: 'd',
+      help: 'Directory name to create project in. Defaults to workspace name',
```

Comment:
```suggestion
      help: 'Directory to create project in. Defaults to workspace name.',
```
Just to conform with the rest of the help hints.